### DISCUSSION TITLE
right-click

### DISCUSSION THREAD

**Discussion by Dan-Essin** [wezterm/wezterm#1221] [Q&A]
Hello,
I really like wezterm but there is one feature I really miss.
I know that you have gone to a lot of trouble to create an elaborate set of keyboard short(?)cuts and multi-click options.
I am so used to getting a right-click menu offering copy and paste that I find the absence of it very jarring and distracting sine it works virtually everywhere else.

Is there something about rust that makes it difficult to implement such a thing?
If not, it would be really helpful if it could be added.
I may take a stab at it but I've never used rust before and I rate my probability of success at <10%

Thank you

**Comment by wez** [comment]
I was thinking about this earlier this week; the challenge is in making it work cross platform rather than it being difficult in rust.
